dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/oracle/V5.7_2017.04.11__DS-3563_Index_metadatavalue_resource_type_id_column.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+------------------------------------------------------
+-- DS-3563 Missing database index on metadatavalue.resource_type_id
+------------------------------------------------------
+-- Create an index on the metadata value resource_type_id column so that it can be searched efficiently.
+
+CREATE INDEX metadatavalue_resource_type_id_idx ON metadatavalue (resource_type_id);
```

Comment:
The Oracle migration should also either check for existence of the index *or* (like Postgres & H2) drop the index if it exists and recreate it. Here's a few resources for how to do that:
* Check if index exists before creation in Oracle: http://openacs.org/xowiki/cond_create_index
* Drop index if exists in Oracle: https://stackoverflow.com/questions/2722630/oracle-drop-index-if-exists